packages/components/addon/components/hds/pagination-bar/pagination.js

```diff
@@ -12,4 +17,34 @@ export default class HdsPaginationBarIndexComponent extends Component {
 
     return type === 'compact';
   }
+
+  get isDisabledPrev() {
+    return this.currentPage === 1;
+  }
+
+  get isDisabledNext() {
+    return this.currentPage === this.totalPages;
+  }
+
+  @action
+  prevPage() {
+    if (this.currentPage > 1) {
+      this.currentPage--;
+      let { pageChanged } = this.args;
+      if (typeof pageChanged === 'function') {
+        pageChanged(this.currentPage);
+      }
+    }
+  }
+
+  @action
+  nextPage() {
```

Comment:
in case you want to avoid the duplication of code between `prevPage` and `nextPage` you could consider to use a generic method `changePage(direction)`, where `direction` (can be `prev` or `next`) controls only the change in the value (+1 or -1)